app/controller/member.controller.js

```diff
@@ -146,3 +147,36 @@ exports.create = async (req, res) => {
     });
 };
 
+
+/*****************************8
+ * Developer: corner
+ * Description: 계정 삭제
+ *             회원의 계정을 삭제합니다.
+ *             params : m_no
+*****************************/
+
+exports.delete = async (req, res) => {
+	if (req.query.hasOwnProperty('m_no') && req.query.m_no === '') {
+		return res.status(200).send({
+			status: 400,
+			message: "Error: m_no값이 없습니다.",
+		});
+	}
+	const m_no = req.query.m_no;
+
+    await Member.destroy({
+        where: {
+            m_no: m_no,
+        }
+    }).then((result) => {
+		if (result === 0) {
+			res.status(402).send({
+				status: 402,
+				message: "해당 회원이 없습니다.",
+			})
+		}
+        res.status(200).send({status: 200, message: "success"});
```

Comment:
Codacy has a fix for the issue: [Replace `········res.status(200).send({status:·200,·message:·"success"` with `↹↹↹res.status(200).send({·status:·200,·message:·"success"·`](https://app.codacy.com/gh/Eight-Corner/oneuleun_web_api/pullRequest?prid=9683172)
```suggestion
			res.status(200).send({ status: 200, message: "success" });
```